packages/map/.vscode/launch.json

```diff
@@ -1,35 +1,60 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "test-browser",
+            "type": "msedge",
+            "request": "launch",
+            "url": "http://localhost:8888",
+            "webRoot": "${workspaceFolder}",
+            "outFiles": [
+                "${workspaceFolder}/**/*.js",
+                "!**/node_modules/**"
+            ],
+        },
+        {
+            "name": "test-node",
+            "type": "node",
+            "request": "launch",
+            "runtimeArgs": [
+                "run-script",
+                "test-node"
+            ],
+            "runtimeExecutable": "npm",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "outFiles": [
+                "${workspaceFolder}/**/*.js",
+                "!**/node_modules/**"
+            ],
+        },
         {
             "name": "index.html",
-            "type": "chrome",
             "request": "launch",
-            "url": "file:///${workspaceRoot}/index.html",
+            "type": "msedge",
+            "url": "file:///${workspaceFolder}/index.html",
             "runtimeArgs": [
                 "--disable-web-security"
             ],
-            "webRoot": "${workspaceRoot}"
+            "webRoot": "${workspaceFolder}",
+            "outFiles": [
+                "${workspaceFolder}/**/*.js",
+                "!**/node_modules/**"
+            ]
         },
         {
-            "name": "Mocha Tests",
-            "type": "pwa-node",
+            "name": "wsl index.html",
             "request": "launch",
-            "program": "${workspaceFolder}/../../node_modules/mocha/bin/_mocha",
-            "args": [
-                "-u",
-                "tdd",
-                "--timeout",
-                "999999",
-                "--colors",
-                "${workspaceFolder}/lib-umd/__tests__"
+            "type": "msedge",
+            "url": "file://wsl.localhost/Ubuntu-24.04/home/gordon/Visualization/packages/map/index.html",
```

Comment:
this was the only reason I mentioned this configuration on that other review, noticing this path to your wsl